lib/reporting/reports/enterprise_fee_summary/enterprise_fees_with_tax_report_by_producer.rb

```diff
@@ -26,9 +24,44 @@ def report_line_items
           @report_line_items ||= Reporting::LineItems.new(order_permissions, params)
         end
 
+        def supplier_ids_filter(supplier_id)
+          return true if params[:supplier_id_in].blank?
+
+          params[:supplier_id_in].include?(supplier_id)
+        end
+
+        def enterprise_fee_filtered_ids
+          return @enterprise_fee_filtered_ids unless @enterprise_fee_filtered_ids.nil?
+
+          @enterprise_fee_filtered_ids = EnterpriseFee.where(nil)
+          unless enterprise_fee_ids_filter.empty?
```

Comment:
I find `unless` had to read here, you could just write the same with a if :
```suggestion
          if enterprise_fee_ids_filter.present?
```